native/src/utils/createSettingsSections.ts

```diff
@@ -71,34 +61,29 @@ const createSettingsSections = ({
               description: t('pushNewsDescription'),
               hasSwitch: true,
               getSettingValue: (settings: SettingsType) => settings.allowPushNotifications,
-              onPress: () => {
-                setSetting(
-                  settings => ({
-                    allowPushNotifications: !settings.allowPushNotifications,
-                  }),
-                  async (newSettings): Promise<boolean> => {
-                    if (!cityCode) {
-                      // No city selected so nothing to do here (should not ever happen since settings are only available from city content routes)
-                      return true
-                    }
+              onPress: async () => {
+                const allowPushNotifications = !settings.allowPushNotifications
```

Comment:
I get linting errors here, that are not on main:
ESLint: Unexpected any value in conditional. An explicit comparison or type cast is required.(@typescript-eslint/strict-boolean-expressions)

Reply:
I cannot reproduce this and it does not happen in the CI, so I am pretty sure that it only happens on your machine. Did you try running `yarn` and `yarn ts:check` before to make sure that all types are up-to-date?

Reply:
![2350-linting](https://github.com/user-attachments/assets/80f7695d-71cf-4642-b066-7515e15732dc)

Did run both commands successfully. Not sure why this is only in Intellij but not in CI